Add a note to Security Authorization doc how to disable the permissions (#13433)
quarkusio/quarkus: docs/src/main/asciidoc/security-authorization.adoc

and JAX-RS endpoints that do not have security annotations but are defined in classes that contain methods with
security annotations. Defaults to `false`.

> === Disabling the permissions 

Reviewer: ```suggestion
=== Disabling permissions 
```

> The permissions can be disabled at build time with a permissions policy specific `enabled` property, for example: 

Reviewer: ```suggestion
Permissions can be disabled at build time with an `enabled` property for each declared permission, for example: 
```
